pkgs/development/python-modules/python-snap7/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildPythonPackage, snap7, fetchPypi, six, setuptools }:
+
+buildPythonPackage rec {
+  pname = "python-snap7";
+  version = "0.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2vg58l8kvy1PE7X+bnuCCKApJInMPkIbt8wd6u0FERA=";
+  };
+
+  buildInput = [ six ];
+  propagatedBuildInputs = [ setuptools ];
+
+  prePatch = ''
+    substituteInPlace snap7/common.py \
+      --replace "lib_location = None" "lib_location = '${snap7}/lib/libsnap7.so'"
+  '';
+
+  # Tests are broken in this release, future release should fix this
+  doCheck = false;
```

Comment:
not.... confidence inducing

 please point to an upstream issue where this is being addressed

Reply:
Ok, looking for what I can do

Reply:
just a reference to an opened issue would suffice, I just want to make sure that upstream is aware of the breakage

Reply:
in fact we will always have a problem, they expect some test to be run as root in order to open privileged tcp port ...

Reply:
oh, then that's fine, just mention that in the comment